lmcache/config.py

```diff
@@ -48,9 +48,12 @@ def from_legacy(
             case "cpu" | "cuda":
                 local_device = backend
                 remote_url = None
+            case path if re.match(r"file://(.*)/", path): #local disk directory
```

Comment:
I think we should also add the regex parsing in `from_file` function below.

Reply:
Oh right. Let me add that real quick